airbyte-integrations/connectors/source-appsflyer/metadata.yaml

```diff
@@ -18,6 +18,11 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      0.2.0:
+        message: Migrate to V2 AppsFlyer API
+        upgradeDeadline: "2024-05-27"
```

Comment:
We will need to bump this when everything else will be ready. Thank you for remembering to add the breaking change notice here!

Reply:
Sure, will bump this when the PR is ready for merging.